src/cace/amm/obj_ns.c

```diff
@@ -143,3 +143,15 @@ cace_amm_obj_desc_t *cace_amm_obj_ns_find_obj_enum(const cace_amm_obj_ns_t *ns,
     }
     return *found;
 }
+
+bool cace_amm_obj_ns_is_odm(const cace_amm_obj_ns_t *ns)
+{
+    CHKFALSE(ns);
+
+    if ((ns->model_id.has_intenum && ns->model_id.intenum < 0) || m_string_start_with_str_p(ns->model_id.name, "!"))
```

Comment:
Recommend to use parentheses around each clause in a condition like this, to be clear to a human (the compiler doesn't care in this case).
```suggestion
    if ((ns->model_id.has_intenum && (ns->model_id.intenum < 0)) || m_string_start_with_str_p(ns->model_id.name, "!"))
```